readme.txt

```diff
@@ -67,7 +67,7 @@ HTTPS Setup Video: [youtube https://www.youtube.com/watch?v=BeTZ2KgytC0]
 
 = 1.17.9 =
 
-- Added 30 second delay to scheduled notification send time, bug fixes
+- Rolled back hard-scheduling feature
```

Comment:
I don't think `hard-scheduling` is clear enough. Let's say something like;
Rollback sending notifications on scheduled posts to be when its status changes to published.